common/src/main/java/org/apache/uniffle/common/util/ThreadUtils.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.uniffle.common.util;
```

Comment:
Could you add license header? Because our project need to follow Apache 2.0 license.

Reply:
done